app/pdfs/certificate_pdf.rb

```diff
@@ -0,0 +1,48 @@
+require 'open-uri'
+
+class CertificatePDF < Prawn::Document
+  attr_accessor :path
+
+  PDF_OPTIONS = {
+    page_size:   'A5',
+    page_layout: :landscape,
+    margin:      [30, 75]
+  }
+
+  def initialize(certificate)
```

Comment:
Method has too many lines. [21/15]

You might want to extract some of this method lines into [new methods](https://sourcemaking.com/refactoring/extract-method) inside its current class or isolate it on a [class of its own](https://sourcemaking.com/refactoring/replace-method-with-method-object) and make it a collaborator instead.